astroid/brain/brain_collections.py

```diff
@@ -77,7 +77,69 @@ def _ordered_dict_mock():
     class OrderedDict(dict):
         def __reversed__(self): return self[::-1]
         def move_to_end(self, key, last=False): pass"""
+    if PY39:
+        base_ordered_dict_class += """
+        @classmethod
+        def __class_getitem__(cls, item): return cls"""
     return base_ordered_dict_class
 
 
-astroid.register_module_extender(astroid.MANAGER, "collections", _collections_transform)
+def _collections_module_properties(node, context=None):
+    """
+    Adds a path to a fictive file as the _collections module is a pure C lib.
+    """
+    node.file = "/tmp/unknown"
```

Comment:
Minor: Could we import the collections module and use it's location to come up with a more realistic location. Maybe `.../_collections.pyi`. Or can we use a placeholder that definitely won't represent a real file. I think we've used `<?>` elsewhere.

Reply:
@AWhetter thanks for your review!
I choose to follow your first advice, so i import the collections module.